Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1348,10 +1347,18 @@ def create_search_with_retry(client: Client, fetch_mode: str, offense: Dict, eve
         f'SELECT {event_columns} FROM events WHERE INOFFENSE({offense_id}) {additional_where} limit {events_limit} '
         f'START {offense_start_time}'
     )
+    print_debug_msg(f'Trying to get events for offense ID: {offense_id}, '
+                    f'query_expression: {query_expression}.')
     num_of_failures = 0
     while num_of_failures <= max_retries:
         try:
-            return client.search_create(query_expression=query_expression)
+            print_debug_msg(f'Creating search for offense ID: {offense_id}, '
+                            f'query_expression: {query_expression}.')
+            ret_value = client.search_create(query_expression=query_expression)
+            print_debug_msg(f'Created search for offense ID: {offense_id}, '
+                            f'query_expression: {query_expression}, '
```

Comment:
same comment as above.

Reply:
Accepted. Changed to: offense_id, offense_start_time, additional_where, events_limit, ret_value.